lib/views/pages/account_settings/account_settings.dart

```diff
@@ -0,0 +1,174 @@
+import 'package:flutter/material.dart';
+import 'package:hooks_riverpod/hooks_riverpod.dart';
+import 'package:sun_flutter_capstone/state/session_provider.dart';
+import 'package:sun_flutter_capstone/views/widgets/template.dart';
+import 'package:sun_flutter_capstone/views/widgets/cards/rectangle.dart';
+
+class AccountSettings extends HookConsumerWidget {
+  const AccountSettings({Key? key}) : super(key: key);
+
+  final String first_name = 'Juan Dela';
+  final String last_name = 'Dela Cruz';
+  final String email = 'juan.delacruz@com.com';
+  final String currency = 'PHP (₱)';
+  final String spending_limit = '18,000';
+
+  @override
+  Widget build(BuildContext context, WidgetRef ref) {
+    final user = ref.watch(sessionProvider);
+
+    return Template(
+        appbarTitle: Text(
+          'Account Settings',
+          style: TextStyle(fontSize: 18, color: Color(0xFF324ED9)),
+        ),
+        content: Container(
+          margin: const EdgeInsets.only(top: 40.0),
+          decoration: new BoxDecoration(
+            borderRadius: BorderRadius.only(
+                topRight: Radius.circular(30.0),
+                topLeft: Radius.circular(30.0)),
```

Comment:
To trigger formatting
```suggestion
                topLeft: Radius.circular(30.0),),
```